### DISCUSSION TITLE
https works in browser, but not for app

### DISCUSSION THREAD

**Discussion by amgschnappi** [immich-app/immich#9248] [Q&A]
### The bug

https works in browser, but not for app

### The OS that Immich Server is running on

Win10

### Version of Immich Server

latest

### Version of Immich Mobile App

latest

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Default
```

### Your .env content

```Shell
Default
```

### Reproduction steps

```bash
1. Using cloudflare tunnel, access http://immich.myweb.com on android app n android firefox browser

2. Using caddy reverse proxy, enable https

3. Https://immich.myweb.com is accessible from firefox andriod

Issue : cannot login to immich app using https
```

**Comment by bo0tzz** [comment]
Please provide more details about your reverse proxy setup. Are you using a self-signed certificate?

**Comment by amgschnappi** [comment]
Cloudflare settings are as below 

![Cloudflare settings](https://github.com/immich-app/immich/assets/102602534/38f272ab-cd31-4437-9689-8857591e32b7)

Caddy file has following 
immich.myweb.com {
    reverse_proxy http://localhost:2283
}

Run caddy.

Https://immich.myweb.com is accessible from firefox andriod

Issue : cannot login to immich app using https

**Comment by ChrisS85** [comment]
I have the same issue, also with caddy, using a certificate signed by LetsEncrypt on a wildcard dynamic dns domain.
What kind of information do you need?

Edit: Forget my report, I had a wrong url with /auth/login at the end.

**Reply by ansh-info** [reply]
Hi @ChrisS85, How did you fix it could you please elaborate. 
Thank you

**Reply by ChrisS85** [reply]
Just what I wrote, I removed /auth/login at the end of the URL. it should be like http(s)://domain-or-ip:port only.

**Reply by ansh-info** [reply]
Thank you for the clarification. I do have one more question: regarding the HTTPS certificate, does it need to be a trusted certificate, or is it acceptable to use a self-signed SSL certificate(like caddy tls)?
